test/e2e/features/story_microshift.feature

```diff
@@ -3,6 +3,7 @@ Feature: Microshift test stories
 
 	Background:
 		Given setting config property "preset" to value "microshift" succeeds
+		And ensuring oc command is available
```

Comment:
This should go as the last step on the background  (cluster should be started fisrt)

Reply:
Yes, I updated it.